README.md

```diff
@@ -18,7 +18,7 @@ Launch Saasform:
 ```bash
 git clone https://github.com/saasform/saasform
 cd saasform
-docker-compose up
+docker-compose up --build
```

Comment:
See next comment

Reply:
I'm approving the PR and will do these update myself, to speed up